x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/middleware.ts

```diff
@@ -43,3 +44,24 @@ export const policyListMiddlewareFactory: MiddlewareFactory<PolicyListState> = c
     }
   };
 };
+
+export const policyDetailsMiddlewareFactory: MiddlewareFactory<PolicyDetailsState> = coreStart => {
```

Comment:
@paul-tavares @parkiino @oatkiller was thinking that we'll keep all state separate from the overall policy list since Policy Details will be it's own route and page.

`ManagementList` keeps the `details` in state under the overall list data because it's a flyout and a query param:
![image](https://user-images.githubusercontent.com/56395104/76460825-23dae580-63b5-11ea-8b3b-cb456966e9fe.png)

Currently, I have PolicyDetails separate in state:
![image](https://user-images.githubusercontent.com/56395104/76460946-5553b100-63b5-11ea-9e10-74d252af9697.png)

If we agree that we should keep `PolicyDetails` separate in state, then I'll go ahead and break out all of this code to be in its own files, etc.  Let me know your thoughts otherwise.

Reply:
I would say 'go w/ it'. Typescript should make refactoring things like this a lot simpler.

Reply:
i think that makes sense too, since it's its own page